app/models.js

```diff
@@ -183,17 +183,17 @@ class Pod extends K8sModel {
     // based on
     // https://github.com/kubernetes/dashboard/blob/91c54261c6a3d7f601c67a2ccfbbe79f3b6a89f9/src/app/frontend/pod/list/card_component.js#L98
 
-    let containerStatus = this.data.status.containerStatuses;
+    const containerStatus = this.data.status.containerStatuses;
 
     if (containerStatus) {
-      let state = containerStatus[0].state;
+      const { state } = containerStatus[0];
```

Comment:
😊  oops

Reply:
No problem, we now we're not 100% linted and that we'll do it "lazily" (as in gradually) :)